Property 'timeLog' is missing in type 'Debugger'

I am getting this error with Angular 6 and "@types/node": "^10.11.4":

ERROR in node_modules/ts-debug/debug.d.ts(1,22): error TS2420: Class 'Debugger' incorrectly implements interface 'Console'.
  Property 'timeLog' is missing in type 'Debugger'

--- a/debug.ts
+++ b/debug.ts
@@ -54,6 +54,10 @@ export class Debugger implements Console {
         return this.doIfEnabled(() => this.console.timeEnd(label));
     }
 
+    public timeLog(label: string, ...data: any[]): void {
+        return this.doIfEnabled(() => this.console.timeLog(label, data));
+    }
+
     public timeStamp(label: string): void;
     public timeStamp(timerName?: string): void;
     public timeStamp(label?: string): void {

